src/core/currency/currency-pixie.ts

```diff
@@ -21,5 +23,28 @@ export const currency: TamePixie<RootProps> = combinePixies({
       walletState: props.state.currency.wallets[walletId],
       walletOutput: props.output.currency.wallets[walletId]
     })
-  )
+  ),
+
+  pluginUpdater(input: ApiInput) {
```

Comment:
When does does this pixie get invoked?

Reply:
The pixie exists during the lifetime of the core, and the returned updater function gets invoked any time Redux changes.